x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/function/FunctionRegistry.java

```diff
@@ -403,6 +403,47 @@ public static FunctionDefinition def(Class<? extends Function> function, Functio
         T build(Source source, Expression src, Expression exp1, Expression exp2, Expression exp3);
     }
 
+
+    // Translation table for error messaging in the following function
+    private static final String[] NUM_NAMES = {
```

Comment:
This should go somewhere at the top of this class.

Reply:
Moved to the top of this class.